src/Test/IntegrationTestCase.php

```diff
@@ -11,69 +11,21 @@
 
 namespace Twig\Test;
 
-use PHPUnit\Framework\TestCase;
-use Twig\Environment;
-use Twig\Error\Error;
-use Twig\Extension\ExtensionInterface;
-use Twig\Loader\ArrayLoader;
-use Twig\RuntimeLoader\RuntimeLoaderInterface;
-use Twig\TwigFilter;
-use Twig\TwigFunction;
-use Twig\TwigTest;
-
 /**
  * Integration test helper.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Karma Dordrak <drak@zikula.org>
+ *
+ * @deprecated use \Twig\Test\TemplateTestCase instead
```

Comment:
The referenced class needs to be updated (but I like this shorter name).

Reply:
I decided against the shorter name because I thought this was too ambiguous in this project.